src/praisonai-agents/tests/unit/test_provider_matrix.py

```
@@ -344,3 +344,56 @@ def upload_file(self, src, dst, timeout=1800):
+    import inspect
+
+    from praisonaiagents.sandbox._sandbox_bridge import resolve_sandbox_class
+
+    sources = []
+    try:
+        from praisonaiagents.managed._compute_bridge import _PROVIDERS
+
+        module, attr = _PROVIDERS[vendor]
+        sources.append(getattr(__import__(module, fromlist=[attr]), attr))
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)